test/Makefile

```
@@ -32,7 +32,7 @@ KUSTOMIZE_DLPATH := $(DOWNLOAD_DIR)/kustomize-v$(KUSTOMIZE_VERSION).tar.gz
 PROMTOOL_DLPATH := $(DOWNLOAD_DIR)/prometheus-v$(PROMTOOL_VERSION).tar.gz
 TELEPORT_DLPATH := $(DOWNLOAD_DIR)/teleport-v$(TELEPORT_VERSION).tar.gz
 
-BINDIR := $(abspath $(PWD)/bin)
+BINDIR = $(abspath $(PWD)/bin)
```

Thank you. CI error is caused by the difference between the `PWD` environment variable and the `CURDIR` Make variable.

```suggestion
BINDIR := $(abspath $(CURDIR)/bin)
```